_posts/en/newsletters/2025-04-04-newsletter.md

```diff
@@ -0,0 +1,569 @@
+---
+title: 'Bitcoin Optech Newsletter #348'
+permalink: /en/newsletters/2025/04/04/
+name: 2025-04-04-newsletter
+slug: 2025-04-04-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- secp256k1lab FIXME:harding
+
+## Changing consensus
+
+_A monthly section summarizing proposals and discussion about changing
+Bitcoin's consensus rules._
+
+- **Multiple discussion about quantum computer theft and resistance:**
+  several conversations examined how Bitcoiners could respond to quantum
+  computers becoming powerful enough to allow stealing bitcoins.
+
+  - *Should vulnerable bitcoins be destroyed?* Jameson Lopp [posted][lopp
+    destroy] to the Bitcoin-Dev mailing list several arguments for the
+    destruction of bitcoins vulnerable to quantum theft after an upgrade
```

Comment:
What follows is an excellent summary of Lopp's post; I'm wondering if we should mention here that his proposal would be a soft fork (he mentions that several times)? Also, it might help future searches of the Optech archive to work in at least some of the words "burn", "unspendable", "freeze", or "frozen" somewhere here.